clupload/cluploadArduino101_osx.sh

```diff
@@ -7,13 +18,21 @@ setup() {
     payload_elf=$2
     # ARG 3: TTY port to use
     tty_port_id=$3
+    # ARG 4: quiet/verbose
+    verbosity=$4
 
     # Upload the .bin instead of .elf
-    payload_bin=${payload_elf/elf/bin}
-    echo "Payload:" $payload_bin
+    payload_bin=${payload_elf/.elf/.bin}
+    dbg_print "Payload:" $payload_bin
 
     export DYLD_LIBRARY_PATH=$dfu_util_dir:$DYLD_LIBRARY_PATH
-    DFU="$dfu_util_dir/dfu-util -d,8087:0ABA"
+    dfu_cmd="$dfu_util_dir/dfu-util -d,8087:0ABA"
+
+    if [ "x$verbosity" == "xverbose" ] ; then
+        dfu_download="$dfu_cmd -D $payload_bin -v --alt 7 -R"
+    else
+        dfu_download="$dfu_cmd -D $payload_bin --alt 7 -R >/dev/null 2>&1"
```

Comment:
Yes, because dfu-utils prints spurious messages to stderr, e.g. "can't detach". We catch the error code, so in case problem occurs, it is handled.